lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -538,6 +541,7 @@ data StakePoolLog
     | MsgStakePoolRegistration PoolRegistrationCertificate
     | MsgRollingBackTo SlotId
     | MsgApplyError ErrMonitorStakePools
+    | MsgUsingRankingEpochConstants Ranking.EpochConstants
```

Comment:
The qualified import isn't needed. Please refer to: https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#we-use-explicit-imports-by-default-and-favor-qualified-imports-for-ambiguous-functions

Reply:
The thing is, I _do_ to some extent view `EpochConstants` as ambiguous and requiring context — `Ranking.EpochContext` — in a similar way as `Map.filter`.

`EpochConstants` are not _all_ epoch-specific constants, but precisely the ones that are needed for `Ranking`. 

(Although,  yes, we actually only have one type called `EpochConstants`.)

Would you still prefer un-qualified?

Reply:
I disagree about the ambiguity here. There's nothing ambiguous about `EpochConstants` here, especially since this identifier only exists once in the entire codebase. On the contrary, if you look at the Registry module for instance, most functions are ambiguous there (`insert`, `remove` etc ...). 

If you think `EpochConstants` lack too much context, then perhaps change the name of the type to `RankingEpochConstants` (although I truly think it's overkill here). 

> Would you still prefer un-qualified?
Yes.